### DISCUSSION TITLE
How do you pin a toot on the Home timeline when running a single user Mastodon instance?

### DISCUSSION THREAD

**Discussion by mtbluedog** [mastodon/mastodon#23537] [General]
Maybe I'm missing something, but how do you pin your own toot on the Home timeline when running a single user Mastodon instance?
Right now, pinning a toot pins it to the top of your profile.
But I want to pin a toot at the top of Home. Is that possible?
Is this not possible because I'm running a single user Mastodon instance?

**Comment by nightpool** [comment]
I don't understand your question. Is this a new feature you're looking to implement? I'm not aware of any feature in mastodon that lets you "pin" a toot to the top of the home timeline. What would be the point of this feature? (especially on a single-user instance)

**Reply by mtbluedog** [reply]
Thanks, I guess it's more like a feature request. I know that Mastodon has Announcements, where you can add one and optionally schedule it, and those are flagged to toggle down to read. I'm thinking more of an announcement or banner that always appears under Home, for user information; or the option to pin an important Toot under Home that always appears there.
